storagenode/bandwidth/usage.go

```diff
@@ -16,12 +16,21 @@ import (
 // architecture: Database
 type DB interface {
 	Add(ctx context.Context, satelliteID storj.NodeID, action pb.PieceAction, amount int64, created time.Time) error
-	// MonthSummary returns summary of the current months bandwidth usages
+	// MonthSummary returns summary of the current months bandwidth usages.
 	MonthSummary(ctx context.Context) (int64, error)
 	Rollup(ctx context.Context) (err error)
+	// Summary returns summary of bandwidth usages.
 	Summary(ctx context.Context, from, to time.Time) (*Usage, error)
+	// EgressSummary returns summary of egress bandwidth usages.
+	EgressSummary(ctx context.Context, from, to time.Time) (*Usage, error)
+	// IngressSummary returns summary of ingress bandwidth usages.
+	IngressSummary(ctx context.Context, from, to time.Time) (*Usage, error)
```

Comment:
I wonder whether these would've been clearer and less requests to the database by having:

```
type Usages struct {
    Bandwidth Usage
    Egress Usage
    Ingress Usage
}

Summary(ctx context.Context, from, to time.Time) (*Usages, error)
```